packages/ng/multi-select/select.validators.ts

```diff
@@ -0,0 +1,13 @@
+import { AbstractControl, ValidationErrors } from '@angular/forms';
+import { LuMultiSelection } from './select.model';
+
+const required = (control: AbstractControl<LuMultiSelection<unknown>>): ValidationErrors | null => {
+	if (control.value && control.value.mode === 'none') {
```

Comment:
```suggestion
	if (!control.value || control.value.mode === 'none') {
```

should we treat `null` value as well?